apps/web/src/pages/index.tsx

```diff
@@ -87,7 +87,7 @@ function Home() {
                 an AI{" "}
                 <Link
                   className="underline"
-                  href="https://github.com/alexanderatallah/Alpaca-Turbo#using-the-api"
+                  href="https://localai.app"
```

Comment:
nit: should we remove www from the other urls?

Reply:
Whoops you're right! - will fix them all to remove the `www`, since that reduce 1 hoop